.github/workflows/integration-test.yml

```diff
@@ -8,12 +8,12 @@ on:
         required: false
         default: ''
       engine:
-        description: 'Engine'
-        required: true
+        description: 'Engine - a new one will be created if not provided'
+        required: false
```

Comment:
This is  somewhat misleading, if database is provided but engine is blank we'll break on `find-engine-name` step. Should we check that both inputs are either defined or undefined?

Reply:
Makes sense, will add this